lib/action_cable/channel/naming.rb

```diff
@@ -0,0 +1,23 @@
+module ActionCable
+  module Channel
+    module Naming
+      extend ActiveSupport::Concern
+
+      class_methods do
+        # Returns the last part of the channels's name, underscored, without the ending
+        # <tt>Channel</tt>. For instance, ChatChannel returns <tt>chat</tt>.
+        #
+        # ==== Returns
+        # * <tt>string</tt>
```

Comment:
Honestly, I'm not sure what you mean by `the last part` because it seems the channel name is scoped after the module namespace (we should also show an example of that).